.expeditor/verify.pipeline.yml

```diff
@@ -571,7 +571,7 @@ steps:
       - .expeditor/scripts/verify/build_package.sh test-services/test-probe
     expeditor:
       executor:
-        linux:
+        docker:
```

Comment:
I think the `env` block needs to be pulled down to an `environment` block under `docker`

Reply:
Hmm, good point.  It does appear to be working though :thinking: 
https://buildkite.com/chef/habitat-sh-habitat-master-verify/builds/5629#3a29f9a0-1008-41ed-a1fd-daaaea8c9cbd/211-603
https://buildkite.com/chef/habitat-sh-habitat-master-verify/builds/5629#46d73875-e5b3-42f4-8278-76c2686e710a/181-524